nutstash-wallet/umbrel-app.yml

```diff
@@ -0,0 +1,31 @@
+manifestVersion: 1
+id: nutstash-wallet
+category: Finance
+name: Nutstash Wallet
+version: "0.1.6"
+tagline: Nutstash is a Cashu wallet for your browser.
+description: >-
+  This app is in early development and most likely buggy. It does handle real sats though, so be sure to use only amounts that are small enough to lose.
+  
+  What is Cashu?
+
+  "Cashu is a free and open-source Chaumian ecash system built for Bitcoin. Cashu offers near-perfect privacy for users of custodial Bitcoin applications. Nobody needs to knows who you are, how much funds you have, and with whom you transact with." - cashu.space
+  
```

Comment:
Here's what this will look like in the app store:

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/85373263/227064368-0d8dee92-cb4e-4536-bcc6-de65feb29dcb.png">

If you want a blank line above `What is Cashu?` then you can add a second newline:

```suggestion
description: >-
  This app is in early development and most likely buggy. It does handle real sats though, so be sure to use only amounts that are small enough to lose.

    
  What is Cashu?

  "Cashu is a free and open-source Chaumian ecash system built for Bitcoin. Cashu offers near-perfect privacy for users of custodial Bitcoin applications. Nobody needs to knows who you are, how much funds you have, and with whom you transact with." - cashu.space
  
```